### PR TITLE
Quotation of variable names needs to be enforeced.

### PR DESCRIPTION
**Pull request by bgruening**
xref: https://github.com/galaxy-iuc/standards/issues/15

### CODE REVIEW

File: docs/best_practices/tool_xml.rst [hunk 1]
```diff
@@ -117,7 +117,7 @@ It's convenient to do something like:
 
 .. code:: console
 
-    ln -sfn $input\_fasta tmp.fa;
+    ln -sfn "${input}"\_fasta tmp.fa;
```
Comment (peterjc) [inline docs/best_practices/tool_xml.rst:120]: Was the original parameter `input` or `input_fasta`? i.e. are the curly braces in the right place?

Also isn't the closing quote in the wrong place? e.g. `ln -sfn "$input\_fasta" tmp.fa;`

Comment (nsoranzo) [inline docs/best_practices/tool_xml.rst:120]: Maybe just remove the `\_fasta`?

Comment (peterjc) [inline docs/best_practices/tool_xml.rst:120]: That would be simpler (although underscores are common in parameter names so having this in the example is quite educational)

Comment (hexylena) [inline docs/best_practices/tool_xml.rst:120]: will get this patched shortly.

**Pull request merged**